airflow/api_connexion/endpoints/user_endpoint.py

```diff
@@ -62,3 +64,43 @@ def get_users(limit, order_by='id', offset=None):
     users = query.offset(offset).limit(limit).all()
 
     return user_collection_schema.dump(UserCollection(users=users, total_entries=total_entries))
+
+
+@security.requires_access([(permissions.ACTION_CAN_CREATE, permissions.RESOURCE_USER)])
+def post_user():
+    """Create a new user"""
+    try:
+        data = user_schema.load(request.json)
+    except ValidationError as e:
+        raise BadRequest(detail=str(e.messages))
+
+    security_manager = current_app.appbuilder.sm
+
+    user = security_manager.find_user(username=data["username"])
+    if user is not None:
+        detail = f"Username `{user.username}` already exists. Please update with PATCH endpoint."
```

Comment:
I just realised I didn’t include a PATCH endpoint 😆 

```suggestion
        detail = f"Username `{user.username}` already exists."
```

Reply:
Should we? Make this PR  be the CUD operations to make the UserAPI complete?